app/boards/shields/leeloo/leeloo_rev2.keymap

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/outputs.h>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+#include <dt-bindings/zmk/rgb.h>
+
+// Layers
+#define DEFAULT    0       // default_layer
+#define LOWER      1       // lower_layer
+#define RAISE      2       // raise_layer
```

Comment:
Same comment as the micro, these can be removed since they aren't used.

Reply:
Thank you @caksoylar!  I removed the unused #define lines.